docker/docker-compose.dev.yml

```diff
@@ -16,44 +16,46 @@ services:
     inventree-dev-server:
         container_name: inventree-dev-server
         build:
-            context: .
+            context: ../
             target: dev
+            dockerfile: docker/Dockerfile
+        environment:
+            INVENTREE_DEV_VENV: "True"
         ports:
             - 8000:8000
+        user: root
         volumes:
-            # Ensure you specify the location of the 'src' directory at the end of this file
-            - src:/home/inventree
+            # Ensure you specify the location of the 'INVENTREE_SRC_DIR' in the dev-config.env file
+            - ${INVENTREE_SRC_DIR:?inventree_src_dir}:/home/inventree
```

Comment:
What is ?inventree_src_dir signifying here?

Reply:
configs like: `${SOMEVAR:?some_missing_var_hint}` will interpolate the SOMEVAR environment variable into the docker-compose.yml file. The ? indicates that the variable is required and has no default in the compose file. If that var is missing, then docker-compose will spit out the hint after the ?.